en/getting_started/sensor_selection.md

```diff
@@ -35,6 +35,15 @@ They are so important because the autopilot does not have other means to detect
 
 For more information and recommended hardware see: [Airspeed Sensors](../sensor/airspeed.md).
 
+## Tachometer
+
+[Revolution-counter sensors](https://en.wikipedia.org/wiki/Tachometer#In_automobiles,_trucks,_tractors_and_aircraft) are *highly recommended* for rotor-wing frames.
+
+They are so important because the autopilot does not have other means to detect stall or another rotor failure. For rotor-wing flight it is the rotation of blades that guarantees lift not airspeed or ground speed.
+
+![Digital RPM sensor](../../images/tachometer_sensor.jpg)
+
+For more information and recommended hardware see: [Tachometer Sensors](../sensor/rpm.md).

```

Comment:
If you look at the docs we typically have a parent page explaining what the sensor does, then "instances" below. Can you create a parent page under sensors for tachometers and then link to that (rather than going straight to your implementation). 
Note, otherwise what happens is that these is no obvious way for people to extend and add other implementations.
The parent page can have most of the same info you have here and any **generic** config for tachometers - though it doesn't look like there is any of that (I assume `PCF8583_POOL` etc are specific to your tachometer?)